src/parser.ts

```diff
@@ -4445,15 +4451,79 @@ export function parseImportExpression(
   if (parser.getToken() === Token.Ellipsis) report(parser, Errors.InvalidSpreadInImport);
 
   const source = parseExpression(parser, context, 1, inGroup, parser.tokenPos, parser.linePos, parser.colPos);
+  let options: ESTree.Expression | null = null;
+
+  if (parser.getToken() === Token.Comma) {
+    consume(parser, context, Token.Comma);
+
+    if (parser.getToken() === Token.LeftBrace) {
+      options = parseExpression(parser, context, 1, inGroup, parser.tokenPos, parser.linePos, parser.colPos);
+
+      if (options && options.type !== 'ObjectExpression') {
```

Comment:
In this case can it become a binding? Anyway the expr parser should detect and throw if missing RightBrace. Meaning this check is both expensive due to perf and not needed

Reply:
That's not latest code.

Reply:
My mistake. Can you link to latest. On cp

Reply:
Just visit the changed code in this PR, you will see latest code.